### PR TITLE
Prometheus: Fix autocomplete does not work on incomplete input

### LINKED ISSUE #20445: Autocomplete does not work on incomplete input
**What happened**:

I type query in explore-loki:
ac

Autocomplete does not happen unless I first close with `"}`

**What you expected to happen**:

Grafana autocompletes my query

**Environment**:
- Grafana version: 6.4.4
- OS Grafana is installed on: linux
- User OS & Browser: linux firefox

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Fixes autocomplete on incomplete input. 
image
image
image

This is unfortunately not going to work for metrics wrapped in functions if the function has correct ending brackets`)`, as we can't determine what is the end of the label. Autocomplete works in a way, where if the value from autocomplete is selected, we rewrite the content of the value up until the end of section/context. And as we don't know what is the end of the label-context is (`}` is missing), we would rewrite the whole end of the query. Therefore this is not implemented in this cases. 

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/20445

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/language_provider.ts [hunk 1]
```diff
@@ -169,6 +169,8 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     const operatorsPattern = /[+\-*/^%]/;
     const isNextOperand = text.match(operatorsPattern);
 
+    console.log(wrapperClasses);
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/prometheus/language_provider.ts:172]:
```suggestion
```

File: public/app/plugins/datasource/prometheus/language_provider.ts [hunk 2]
```diff
@@ -169,6 +169,7 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     const operatorsPattern = /[+\-*/^%]/;
     const isNextOperand = text.match(operatorsPattern);
 
+
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/prometheus/language_provider.ts:172]:
```suggestion
```

**Review by zoltanbedi — APPROVED** [review]
While I'm not the best at regexes I tried it out and it works now so looks good 🍏

**Comment by roidelapluie** [comment]
Thanks!!!

**Pull request merged**